examples/access-token-server/package.json

```diff
@@ -11,7 +11,7 @@
     "create-status-json": "ts-node script/create_status_json.ts"
   },
   "dependencies": {
-    "@looker/sdk": "^0.3.7-beta.0",
+    "@looker/sdk": "^0.3.8-beta.0",
```

Comment:
Should this be updated to 7.2.0 and include the rtl?

Reply:
Let's make that another PR